examples/active-active/main.tf

```diff
@@ -0,0 +1,101 @@
+provider "aws" {
+  default_tags {
+    tags = var.tags
+  }
+  region = "us-east-2"
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+}
+
+resource "random_string" "friendly_name" {
+  length  = 4
+  upper   = false # Some AWS resources do not accept uppercase characters.
+  number  = false
+  special = false
+}
+
+# Keypair for SSH
+# ---------------
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+}
+
+resource "local_file" "private_key_pem" {
+  filename = "${path.module}/work/private-key.pem"
+
+  content         = tls_private_key.main.private_key_pem
+  file_permission = "0600"
+}
+
+resource "aws_key_pair" "main" {
+  public_key = tls_private_key.main.public_key_openssh
+
+  key_name = "${local.friendly_name_prefix}-ssh"
+}
+
+# Key Management Service
+# ----------------------
+
```

Comment:
Can you please remove the extra white space after the title sections for consistency?